src/umockdev-record.vala

```diff
@@ -430,6 +454,16 @@ main (string[] args)
         preload = preload + ":";
     Environment.set_variable("LD_PRELOAD", preload + "libumockdev-preload.so.0", true);
 
+    try {
+        root_dir = DirUtils.make_tmp("umockdev.XXXXXX");
+        Environment.set_variable("UMOCKDEV_DIR", root_dir, true);
+    } catch (FileError e) {
+        error("Cannot create temporary directory: %s", e.message);
+    }
+
+    // "disable" the testbed, the library will still try to load the ioctl
+    FileStream.open(Path.build_filename(root_dir, "disabled"), "w");
```

Comment:
This should be closed again.

Reply:
This happens implicitly as the returned `FILE*` is immediately out of scope.